db/migrate/20170216193758_add_vm_table_comments.rb

```diff
@@ -0,0 +1,81 @@
+class AddVmTableComments < ActiveRecord::Migration[5.0]
+  def change
+    change_table_comment :vms, "Virtual Machines"
+
+    change_column_comment :vms, :autostart, "A boolean value that indicates whether or not the VM is set to autostart."
+    change_column_comment :vms, :availability_zone_id, "FK to the AVAILABILITY_ZONES table."
+    change_column_comment :vms, :busy, "HELP"
+    change_column_comment :vms, :boot_time, "The last time the VM was started."
+    change_column_comment :vms, :cloud, "HELP"
+    change_column_comment :vms, :cloud_network_id, "FK to the CLOUD_NETWORKS table."
+    change_column_comment :vms, :cloud_subnet_id, "FK to the CLOUD_SUBNETS table."
+    change_column_comment :vms, :cloud_tenant_id, "FK to the CLOUD_TENANTS table."
+    change_column_comment :vms, :config_xml, "HELP"
+    change_column_comment :vms, :connection_state, "HELP"
+    change_column_comment :vms, :cpu_affinity, "HELP"
+    change_column_comment :vms, :cpu_hot_add_enabled, "HELP"
+    change_column_comment :vms, :cpu_hot_remove_enabled, "HELP"
+    change_column_comment :vms, :cpu_limit, "HELP"
+    change_column_comment :vms, :cpu_reserve, "HELP"
+    change_column_comment :vms, :cpu_reserve_expand, "HELP"
+    change_column_comment :vms, :cpu_shares, "HELP"
+    change_column_comment :vms, :cpu_shares_level, "HELP"
+    change_column_comment :vms, :created_on, "The date the VM was added to ManageIQ inventory."
```

Comment:
Do not use `ManageIQ` anywhere in the descriptions because a) it's redundant and b) we also have CloudForms